src/test/java/core/basesyntax/service/FruitShopServiceTest.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.impl.FruitShopServiceImpl;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import core.basesyntax.strategy.operation.BalanceOperationHandlerImpl;
+import core.basesyntax.strategy.operation.OperationHandler;
+import core.basesyntax.strategy.operation.PurchaseOperationHandlerImpl;
+import core.basesyntax.strategy.operation.ReturnOperationHandlerImpl;
+import core.basesyntax.strategy.operation.SupplyOperationHandlerImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitShopServiceTest {
+    private static final Map<FruitTransaction.Operation,
+            OperationHandler> STRATEGIES = new HashMap<>();
+    private static OperationStrategy operationStrategy;
+    private static FruitShopServiceImpl fruitShopService;
+
+    @BeforeClass
+    public static void beforeClass() {
+        STRATEGIES.put(FruitTransaction.Operation.BALANCE, new BalanceOperationHandlerImpl());
+        STRATEGIES.put(FruitTransaction.Operation.SUPPLY, new SupplyOperationHandlerImpl());
+        STRATEGIES.put(FruitTransaction.Operation.PURCHASE, new PurchaseOperationHandlerImpl());
+        STRATEGIES.put(FruitTransaction.Operation.RETURN, new ReturnOperationHandlerImpl());
```

Comment:
```suggestion
        Map<FruitTransaction.Operation, OperationHandler> handlersMap = Maps.of(FruitTransaction.Operation.BALANCE, new BalanceOperationHandlerImpl(),
        FruitTransaction.Operation.SUPPLY, new SupplyOperationHandlerImpl(),
        FruitTransaction.Operation.PURCHASE, new PurchaseOperationHandlerImpl(),
        FruitTransaction.Operation.RETURN, new ReturnOperationHandlerImpl());
```
this can be local variable